src/core_modules/capture-core/components/Pagination/withDefaultNavigation.js

```diff
@@ -73,19 +67,11 @@ const getNavigation = (InnerComponent: React.ComponentType<any>) =>
                     <IconButton
                         data-test={'dhis2-capture-search-pagination-next-page'}
                         onClick={this.handleNextButtonClick}
-                        disabled={currentPage >= Math.ceil(rowsCount / rowsPerPage)}
```

Comment:
Well this is a bit problematic. Perhaps you have some ideas how to achieve the same.

Reply:
I am passing the `isPaginationButtonDisabled` for the Pagination. This way the one who is using the component can feed it with this property accordingly or not.